apps/web/src/server/applications/case/__tests__/applications-case.test.js

```diff
@@ -222,13 +222,5 @@ describe('Applications case pages', () => {
 				expect(element.innerHTML).toContain('Unpublish the project');
 			});
 		});
-
-		describe('POST /case/123/unpublish', () => {
```

Comment:
why was this removed?

Reply:
It was failing for a weird reason. I've added it back in and fixed the issue now.